src/components/PasswordComplexity.tsx

```diff
@@ -11,22 +11,22 @@ const PasswordComplexityDiv = styled.div`
   gap: 4px;
 `;
 
-const PasswordRequirementDiv = styled.div<{ met: boolean }>`
+const PasswordRequirementDiv = styled.div`
   display: flex;
   align-items: center;
   gap: 8px;
 `;
 
-const PasswordRequirementText = styled.p<{ met: boolean }>`
+const PasswordRequirementText = styled.p<{ $met: boolean }>`
   ${sans.style}
-  color: ${props => (props.met ? COLORS.green : COLORS.greyMid)};
+  color: ${props => (props.$met ? COLORS.green : COLORS.greyMid)};
```

Comment:
should we use destructuring here for consistency?

Reply:
i destructured i am so cool